### PR TITLE
Introduce more calls to `gear-node-loader` (loader MVP-version)

### LINKED ISSUE #1449: Implement all `pallet_gear` extrinsics generation and test
Preferably after #1448. That requires implementing:

MVP
- [x] #1601

Phase 2: 
- [ ] send_reply
- [ ] claim_value

Note: these extrinsics require some additional context to be introduced during tests generation.

### PR DESCRIPTION
**Pull request by SabaunT**
Partially resolved #1449 .

- Implements send_message test task
- Implements upload_code test task
- Implements create_program test task

The functionality can be considered as an MVP version of the `gear-node-loader`.

### CODE REVIEW

**Comment by SabaunT** [comment]
I know there's plenty of copy paste, but that's required to be cleaned in #1600

File: gclient/src/api/calls.rs [hunk 1]
```diff
@@ -266,7 +266,7 @@ impl GearApi {
     pub async fn send_message_bytes_batch(
         &self,
         args: impl IntoIterator<Item = (ProgramId, impl AsRef<[u8]>, u64, u128)>,
-    ) -> Result<(Vec<Result<MessageId>>, H256)> {
+    ) -> Result<(Vec<Result<(MessageId, ProgramId)>>, H256)> {
```
Comment (breathx) [inline gclient/src/api/calls.rs:269]: Why do we need this thing?
Any data we own before fn call shouldn't be returned as the result for me

File: utils/node-loader/src/batch_pool/batch/upload_code.rs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+pub struct UploadCodeArgs(pub Vec<u8>);
+
+impl From<UploadCodeArgs> for Vec<u8> {
+    fn from(UploadCodeArgs(code): UploadCodeArgs) -> Self {
+        code
+    }
+}
+
+impl UploadCodeArgs {
+    pub fn generate<Rng: LoaderRng>(code_seed: Seed) -> Self {
```
Comment (breathx) [inline utils/node-loader/src/batch_pool/batch/upload_code.rs:17]: Btw for args generation may be some trait introduced (comment just for future)

**Review by breathx — APPROVED** [review]

File: utils/node-loader/src/batch_pool/batch/upload_code.rs (hunk 2 above)
Comment (SabaunT) [reply utils/node-loader/src/batch_pool/batch/upload_code.rs:17]: will bear in mind

**Pull request merged**